templates/partial/_footer.html

```diff
@@ -0,0 +1,24 @@
+<footer class="p-strip">
```

Comment:
Indentation should be 2 spaces everywhere.

Reply:
Done